frontend/src/routes/main.tsx

```diff
@@ -6,15 +6,18 @@ import {
   LoginPage,
   LogoutPage,
   PasswordForgottenPage,
-  PasswordResetPage
+  PasswordResetPage,
+  PrivateRoute,
 } from 'auth/components';
 import { RegistrationContainer } from 'registration/components';
 import { ConsoleContainer } from 'console/components';
 import { ROUTES } from '../global/constants';
 
 export const MainRoutes = () => (
   <Switch>
-    <Redirect from="/" to={ROUTES.Registration.HOME} exact />
+    <PrivateRoute exact path="/" ifUnauthorizedRedirectTo={ROUTES.Registration.HOME}>
+      <Redirect to={ROUTES.Console.HOME} />
+    </PrivateRoute>
```

Comment:
This will take you to the Console home if you are logged in, but take you to the registration home if you are logged out.

Reply:
@xitij2000 Can you change the console home to the `Theme preview and colors` page on https://github.com/open-craft/opencraft/blob/6df9c2e14cf0b859159fc847dcf0d820a8576fc0/frontend/src/routes/console.tsx#L22?